orangecontrib/text/widgets/icons/Scraper.svg

```diff
@@ -0,0 +1,54 @@
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="512.000000pt" height="512.000000pt" viewBox="0 0 512.000000 512.000000"
+ preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.13, written by Peter Selinger 2001-2015
+</metadata>
+<g transform="translate(0.000000,512.000000) scale(0.100000,-0.100000)"
+fill="#000000" stroke="none">
+<path d="M1403 5110 c-118 -24 -236 -118 -294 -235 -34 -69 -34 -70 -37 -232
+l-4 -163 -338 0 c-213 0 -359 -4 -392 -11 -153 -33 -283 -158 -324 -313 -21
+-79 -20 -3550 1 -3650 50 -239 252 -441 491 -491 99 -21 4211 -22 4289 -1 153
+40 271 158 311 311 21 75 20 4396 0 4471 -41 155 -171 280 -324 313 -63 13
+-3311 14 -3379 1z m3335 -211 c59 -12 121 -61 148 -116 18 -37 19 -107 22
+-2193 2 -1503 -1 -2171 -8 -2207 -8 -36 -23 -65 -51 -97 -73 -82 84 -76 -1936
+-76 l-1793 0 40 59 c52 75 89 160 106 241 11 54 14 414 14 2146 l0 2080 26 53
+c29 58 82 100 142 112 62 12 3234 11 3290 -2z m-3670 -2429 c1 -1029 -1 -1828
+-7 -1872 -25 -212 -180 -365 -389 -380 -226 -18 -424 144 -453 370 -13 105
+-11 3450 2 3513 15 69 85 142 150 157 25 6 186 9 369 8 l325 -1 3 -1795z"/>
+<path d="M1663 4255 c-33 -14 -63 -59 -63 -95 0 -37 30 -81 65 -96 51 -21
+1099 -21 1150 0 35 15 65 59 65 96 0 37 -30 81 -65 96 -50 21 -1104 20 -1152
+-1z"/>
+<path d="M3383 4263 c-12 -2 -34 -18 -48 -34 -36 -43 -34 -98 4 -141 l29 -33
+549 -3 c301 -2 562 0 579 3 39 7 81 47 89 85 9 41 -19 95 -60 114 -29 14 -100
+16 -577 15 -298 -1 -553 -4 -565 -6z"/>
+<path d="M1672 3619 c-36 -11 -72 -59 -72 -97 0 -39 29 -83 65 -98 51 -21
+1099 -21 1150 0 36 15 65 59 65 98 0 41 -36 86 -77 97 -48 14 -1084 13 -1131
+0z"/>
+<path d="M3272 3619 c-20 -6 -40 -22 -52 -42 -19 -31 -20 -54 -20 -695 0 -736
+-2 -710 65 -738 52 -22 1323 -21 1365 1 63 33 60 -7 60 736 0 666 0 677 -20
+702 -12 15 -38 31 -58 37 -48 13 -1294 12 -1340 -1z m1208 -739 l0 -530 -535
+0 -535 0 0 530 0 530 535 0 535 0 0 -530z"/>
+<path d="M1672 2979 c-36 -11 -72 -59 -72 -97 0 -39 29 -83 65 -98 51 -21
+1099 -21 1150 0 36 15 65 59 65 98 0 41 -36 86 -77 97 -48 14 -1084 13 -1131
+0z"/>
+<path d="M1663 2335 c-33 -14 -63 -59 -63 -95 0 -37 30 -81 65 -96 51 -21
+1099 -21 1150 0 35 15 65 59 65 96 0 37 -30 81 -65 96 -50 21 -1104 20 -1152
+-1z"/>
+<path d="M1663 1695 c-33 -14 -63 -59 -63 -95 0 -37 30 -81 65 -96 51 -21
+1099 -21 1150 0 35 15 65 59 65 96 0 37 -30 81 -65 96 -50 21 -1104 20 -1152
+-1z"/>
+<path d="M3383 1703 c-12 -2 -34 -18 -48 -34 -36 -43 -34 -98 4 -141 l29 -33
+549 -3 c301 -2 562 0 579 3 39 7 81 47 89 85 9 41 -19 95 -60 114 -29 14 -100
+16 -577 15 -298 -1 -553 -4 -565 -6z"/>
+<path d="M1663 1055 c-33 -14 -63 -59 -63 -95 0 -37 30 -81 65 -96 51 -21
+1099 -21 1150 0 35 15 65 59 65 96 0 37 -30 81 -65 96 -50 21 -1104 20 -1152
+-1z"/>
+<path d="M3383 1063 c-12 -2 -34 -18 -48 -34 -36 -43 -34 -98 4 -141 l29 -33
+549 -3 c301 -2 562 0 579 3 39 7 81 47 89 85 9 41 -19 95 -60 114 -29 14 -100
+16 -577 15 -298 -1 -553 -4 -565 -6z"/>
+</g>
+</svg>
```

Comment:
Comparing to other icons, this one seem too big. Once on the canvas, it is touching the edge of the circle, while other widgets have some space between the icon and the circle border. Can you please make it smaller?